src/installing-workflow/configuring-object-storage.md

```diff
@@ -32,6 +32,7 @@ The helm classic chart for Deis Workflow can be easily configured to connect Wor
             ```
             export AWS_ACCESS_KEY, AWS_SECRET_KEY, AWS_REGISTRY_BUCKET, AWS_DATABASE_BUCKET, AWS_BUILDER_BUCKET, S3_REGION
             ```
+              * To use [IAM credentials][aws-iam], it is not necessary to set AWS_ACCESS_KEY or AWS_SECRET_KEY.
```

Comment:
can you put "AWS_ACCESS_KEY" and "AWS_SECRET_KEY" in tick marks so that they show up as `pre-formatted`:

```console
..., it is not necessary to set `AWS_ACCESS_KEY` or `AWS_SECRET_KEY`.
```